conans/client/command.py

```diff
@@ -1076,17 +1171,17 @@ def main(args):
     parse parameters
     """
     try:
-        conan_api = Conan.factory()
+        conan_api, client_cache, user_io = Conan.factory()
```

Comment:
Just linter, not accessing private members.

Reply:
Ok